### ISSUE TITLE
Deep Learning in Label-free Cell Classification

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#31]
https://dx.doi.org/10.1038/srep21471

**Comment by cgreene** [comment]
Thinking about including this in categorize. Worth noting that the features are expert designed before being fed into the NN. Since NNs are generally quite good at the feature construction stage - particularly in images - maybe this is a lost opportunity.

**Comment by agitter** [comment]
I didn't read this one yet.  Are they manually constructing features because there is not enough training data?  That would relate to the discussion in #95.

**Comment by cgreene** [comment]
I didn't see a reason specified. They also use only 16 features from my read:
http://www.nature.com/articles/srep21471/tables/1

This is discussed in the "Feature Extraction" section.

**Comment by cgreene** [comment]
From our intro:

> For the purposes of this review, we identify deep learning approaches as those that use multi-layer neural networks to construct complex features from large-scale datasets.

Since there are 16 manually constructed features here, I'm not sure this quite fits our bill. I'm going to close for now. If someone wants to discuss, feel free to re-open.
